Extensions/TextObject/Extension.cpp

```diff
@@ -566,8 +569,10 @@ void DeclareTextObjectExtension(gd::PlatformExtension& extension) {
   obj.AddDuplicatedAction("Size", "Text::SetFontSize").SetHidden();
   obj.AddDuplicatedCondition("Size", "Text::FontSize").SetHidden();
 
+  // Deprecated
   obj.AddStrExpression(
```

Comment:
I can't help but think that it would be worth keeping it - even if it's redundant.
`Score.String()` is nice. `Score.Text::Value()` is ok, but a bit less nice.

Reply:
Not ideal but let's try this.